iac-extensions/docker/src/test/resources/checks/SpecificVersionTagCheck/specificVersionTagCheck.dockerfile

```diff
@@ -0,0 +1,79 @@
+ARG IMAGE_NAME=my-image
+ARG IMAGE_WITH_TAG=my-image:1.2.3
+ARG LATEST_TAG=latest
+ARG NON_LATEST_TAG=1.2.3
+
+# Noncompliant@+1
+FROM --foo=bar foobar
+
+# Noncompliant@+1 {{Use a specific version tag for the image.}}
+FROM my-image
+#    ^^^^^^^^
+# Noncompliant@+1
+FROM my-image:latest
+#    ^^^^^^^^^^^^^^^
+
+# Noncompliant@+1
+FROM foobar
+# Noncompliant@+1
+FROM foobar:latest
+
+# Noncompliant@+1
+FROM --platform=foo bar
+# Noncompliant@+1
+FROM foobar AS fb
+# Noncompliant@+1
+FROM --platform=foo bar AS fb
+# Noncompliant@+1
+FROM foobar:latest AS fb
+# Noncompliant@+1
+FROM --platform=foo bar:latest
+# Noncompliant@+1
+FROM --platform bar:latest
+# Noncompliant@+1
+FROM --platform=foo bar:latest AS fb
+# Noncompliant@+1
+FROM foobar:latest as baseWithoutTag
+
+# Noncompliant@+1
+from foobar
+
+# Noncompliant@+1
+FROM ${IMAGE_NAME}:${LATEST_TAG}
+#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# Noncompliant@+1
+FROM ${IMAGE_NAME}
+#    ^^^^^^^^^^^^^
+
+# Noncompliant@+2
+FROM \
+foobar
+
+# Noncompliant@+2
+FROM \
+foobar:latest
+
+# Compliant
+FROM my-image:1.2.3 as baseWithTag
+FROM my-image:1.2.3-alpine
+FROM ubuntu:18.04 as local-toolchain-ubuntu18.04-manylinux2010
+FROM foobar@12313423 # Compliant usage of digest
```

Comment:
I didn't saw in the ticket nor in the rspec PR that using a digest is considered as compliant, to me we should still raise an issue in that case.

Reply:
Well a digest is not a version tag, so we shouldn't raise here.
The ticket only describes to raise an issue on a missing version tag.
And it wouldn't make sense to raise an issue here on every digest, because we shouldn't discourage from using it.